packages/salesforcedx-vscode-core/test/vscode-integration/orgBrowser/metadataType.test.ts

```diff
@@ -18,30 +18,24 @@ import { OrgAuthInfo, workspaceUtils } from '../../../src/util';
 describe('get metadata types folder', () => {
   let getDefaultUsernameStub: SinonStub;
   let getUsernameStub: SinonStub;
+  let metadataFolderStub: SinonStub;
   const rootWorkspacePath = workspaceUtils.getRootWorkspacePath();
+  const metadataDirectoryPath = 'test/path/.sfdx/orgs/test-username1@example.com/metadata';
   const typeUtil = new TypeUtils();
   beforeEach(() => {
     getDefaultUsernameStub = stub(OrgAuthInfo, 'getDefaultUsernameOrAlias');
     getUsernameStub = stub(OrgAuthInfo, 'getUsername');
+    metadataFolderStub = stub(projectPaths, 'metadataFolder').returns(
+      metadataDirectoryPath
+    );
   });
   afterEach(() => {
     getDefaultUsernameStub.restore();
     getUsernameStub.restore();
   });
 
```

Comment:
I think if you write some jest unit tests for this method we can just remove this int test as it's not really an int test anyway.